app/views/generic_files/upload/_uploadlimit.html.erb

```diff
@@ -0,0 +1,3 @@
+<label>
+  <strong> Upload limit: Maximum upload size is limited to 200MB each file. Maximum upload total size is limited to 500MB. Maximum number of files allowed to be uploaded is 100.</strong>
```

Comment:
I would rewrite this as "Upload limits: You may upload up to 100 files at a time. Each file cannot be larger than 200 MBs. Your upload cannot be larger than 500 MBs in total."